packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -264,15 +266,17 @@ export default class ListControl extends React.Component {
     const items = value || List();
     const label = field.get('label');
     const labelSingular = field.get('label_singular') || field.get('label');
-    const listLabel = items.size === 1 ? labelSingular.toLowerCase() : label.toLowerCase();
+    const listLabel = items.size < 2 ? labelSingular.toLowerCase() : label.toLowerCase();
```

Comment:
This change is incorrect. There are 0 cookies in the jar is the correct way to say it, as it was in the earlier implementation. When there is 0 in a collection, the plural is used.

Reply:
It is surprising to see how languages work differently, I reverted it.